### PR TITLE
Separate the release of bigdl-tf and bigdl-math

### PR DESCRIPTION
**Pull request by hkvision**

### CODE REVIEW

File: python/tflibs/dev/release.sh [hunk 1]
```diff
@@ -60,8 +60,8 @@ elif [ "$platform" == "linux" ]; then
     mv "${FRAMEWORK_SO}" "${SO_PREFIX}/libtensorflow_framework-zoo.so"
     # Need root permission to execute the following commands.
     # Add sudo to these two commands if you are not the root.
-    patchelf --set-soname libtensorflow_framework-zoo.so ${SO_PREFIX}/libtensorflow_framework-zoo.so
-    patchelf --replace-needed libtensorflow_framework.so.1 libtensorflow_framework-zoo.so ${SO_PREFIX}/libtensorflow_jni.so
+    sudo patchelf --set-soname libtensorflow_framework-zoo.so ${SO_PREFIX}/libtensorflow_framework-zoo.so
+    sudo patchelf --replace-needed libtensorflow_framework.so.1 libtensorflow_framework-zoo.so ${SO_PREFIX}/libtensorflow_jni.so
```
Comment (hkvision) [inline python/tflibs/dev/release.sh:64]: revert

**Comment by hkvision** [comment]
Tested locally.

**Comment by yushan111** [comment]
A bunch of Jenkins job fails due to the renaming.

For example, in http://10.112.231.51:18888/job/ZOO-PR-BigDL-Python-Spark-2.4-py36-ray/91/console
```
/tmp/jenkins4139263618913106920.sh: line 46: ./python/orca/dev/release/all_release_default_linux_spark246.sh: No such file or directory
```

Please make sure Jenkins jobs pass before merging PR.

**Comment by hkvision** [comment]
You shouldn't call the all_release script in jenkins. For all jobs, you need to use the combined script.

**Pull request merged**